sample-node-config/aws/scripts/login.sh

```diff
@@ -7,7 +7,9 @@ set -e
 [ -z "${AWS_PROFILE}" ] && { echo "please set env variable AWS_PROFILE"; exit 1; }
 
 CONNECT_AS=${1:-'ubuntu'}
-KEY_PAIR_LOCATION=${2:-'./env/personal.pem'}
+
+# TODO: the key name is hardcoded here, can we do better?
+KEY_PAIR_LOCATION=${2:-'./env/hydra.pem'}
```

Comment:
This is an SSH key right? `hydra.pem` is a bit misleading as we are also dealing with hydra signing/verification keys when running a hydra-node.

Reply:
My bad, I wasn't suppose to commit this line.

Reply:
reverted until we can make all these scripts to run parametrized or automated